src/__tests__/test-helpers/bridge.js

```diff
@@ -425,6 +425,38 @@ export function testBridge<T>(family: string, data: DatasetTest<T>) {
               }
             });
 
+            makeTest("account always has tronResources", async () => {
+              const { tronResources = {} } = await getSynced();
+
+              expect(tronResources).toEqual(
+                expect.objectContaining({
+                  votes: expect.any(Array),
+                  tronPower: expect.any(Number),
+                  energy: expect.toBeBigNumber(),
+                  bandwidth: {
+                    freeUsed: expect.toBeBigNumber(),
+                    freeLimit: expect.toBeBigNumber(),
+                    gainedUsed: expect.toBeBigNumber(),
+                    gainedLimit: expect.toBeBigNumber(),
+                  },
+                  unwithdrawnReward: expect.toBeBigNumber(),
+                  cacheTransactionInfoById: expect.any(Object),
+                })
+              );
+              expect([
+                "bandwidth",
+                "cacheTransactionInfoById",
+                "delegatedFrozen",
+                "energy",
+                "frozen",
+                "lastVotedDate",
+                "lastWithdrawnRewardDate",
+                "tronPower",
+                "unwithdrawnReward",
+                "votes",
+              ]).toEqual(expect.arrayContaining(Object.keys(tronResources)));
+            });
```

Comment:
Jest has really been a pain in the $$$ to make what looked like a pretty simple test "Does tronResources have all this properties/type combo and some can be undefined". 

That's the best thing I could find without risking 10 sick days for mental breakdown.